library/src/main/scala-2.10_2.13.0-M3/scala/scalajs/ScalaVersionSpecificRuntime.scala

```diff
@@ -0,0 +1,19 @@
+package scala.scalajs
+
+import scala.collection.GenTraversableOnce
+
+trait ScalaVersionSpecificRuntime {
```

Comment:
This trait contains the parts of `scala.scalajs.Runtime` that are specific to Scala for versions 2.10 to 2.13.0-M3.

I’ve used the same technique for some other files. Alternatively, when there were too many differences, I duplicated the complete implementation.